### PR TITLE
Bug 1585731 fix 3.7 local volume procedure

### PR DESCRIPTION
**Pull request by bfallonf**
For: https://bugzilla.redhat.com/show_bug.cgi?id=1585731

Will verify in the BZ.

cc: @jsafrane

### CODE REVIEW

**Comment by jsafrane** [comment]
lgtm from engineering point of view

**Comment by bfallonf** [comment]
Thanks @jsafrane 

@openshift/team-documentation PTAL

File: install_config/configuring_local.adoc [hunk 1]
```diff
@@ -71,13 +71,21 @@ a disk partition or an LVM), create suitable file systems on these devices, and
...
-/dev/sdc1       /mount/local-storage/hdd/disk1 ext4     defaults 1 2
-/dev/sdc2       /mount/local-storage/hdd/disk2 ext4     defaults 1 2
+/dev/sdb1       /mnt/local-storage/ssd/disk1 ext4     defaults 1 2
+/dev/sdb2       /mnt/local-storage/ssd/disk2 ext4     defaults 1 2
+/dev/sdb3       /mnt/local-storage/ssd/disk3 ext4     defaults 1 2
+/dev/sdc1       /mnt/local-storage/hdd/disk1 ext4     defaults 1 2
+/dev/sdc2       /mnt/local-storage/hdd/disk2 ext4     defaults 1 2
 ----
 
+All volumes must be accessible to processes running within Docker containers.
```
Comment (kalexand-rh) [inline install_config/configuring_local.adoc]: s/running/that run

File: install_config/configuring_local.adoc [hunk 2]
```diff
@@ -71,13 +71,21 @@ a disk partition or an LVM), create suitable file systems on these devices, and
...
+Change the labels of mounted filesystems to allow that:
```
Comment (kalexand-rh) [inline install_config/configuring_local.adoc]: I'd s/allow that/grant this access

**Review by kalexand-rh — COMMENTED** [review]
A couple of picks to consider, otherwise LGTM

File: install_config/configuring_local.adoc (hunk 2 above)
Comment (bfallonf) [reply install_config/configuring_local.adoc]: Edited the two sentences together.

**Comment by bfallonf** [comment]
Thanks @kalexand-rh . I'll merge.

**Pull request merged**